src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/">Home</Link>
```

Comment:
Always look to clean up the code, it's much easier to map through an array of like items than it is to write it over and over when they're going to be the same. Try to go back and refactor your code when you have the time.